app/models/ocelot/Meta.scala

```diff
@@ -19,7 +19,18 @@ package models.ocelot
 import play.api.libs.json._
 import play.api.libs.functional.syntax._
 
-case class Meta(id: String, title: String, ocelot: Int, lastAuthor: String, lastUpdate: Long, version: Int, fileName: String, titlePhrase: Option[Int] = None)
+case class Meta(id: String,
+                title: String,
+                ocelot: Int,
+                lastAuthor: String,
+                lastUpdate: Long,
+                version: Int,
+                fileName: String,
+                titlePhrase: Option[Int] = None,
+                processCode: Option[String] = None) {
+
+  def getProcessIdentifier(): String = processCode.getOrElse(id)
```

Comment:
lazy val as we only need it the check executed once

Reply:
Added update